go-controller/go.mod

```diff
@@ -45,6 +47,7 @@ require (
 
 replace (
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/ovn-org/libovsdb => github.com/dave-tucker/libovsdb v0.3.2-0.20210527154300-08498f8ac6ae
```

Comment:
is this just for testing?

Reply:
yep, until we tag v0.4.0, but that'  won't happen until some time next week